testing/jest/button.tests.tsx

```diff
@@ -79,6 +79,39 @@ describe('Button', () => {
                 expect(buttonContentChildren.render().text()).toBe('My button123');
             });
         });
+
+        describe('iconPosition', () => {
+            it('should render icon before text if iconPosition is left', () => {
```

Comment:
Also, the default behavior should be tested. Or what right default properties are defined.